backend/typescript/rest/userRoutes.ts

```diff
@@ -25,64 +25,68 @@ const authService: IAuthService = new AuthService(userService, emailService);
 
 // ROLES: Admin
 /* Get all users, optionally filter by a userId or email query parameter to retrieve a single user */
-userRouter.get("/", async (req, res) => {
-  const { userId, email } = req.query;
-  const contentType = req.headers["content-type"];
-
-  if (userId && email) {
-    await sendResponseByMimeType(res, 400, contentType, [
-      {
-        error: "Cannot query by both userId and email.",
-      },
-    ]);
-    return;
-  }
-
-  if (!userId && !email) {
-    try {
-      const users = await userService.getUsers();
-      await sendResponseByMimeType<UserDTO>(res, 200, contentType, users);
-    } catch (error: unknown) {
-      await sendResponseByMimeType(res, 500, contentType, [
+userRouter.get(
+  "/",
+  isAuthorizedByRole(new Set(["Admin"])),
```

Comment:
@dhmi19 Can camp coordinators see the camp details page? Because i think this call is part of that page loading

Reply:
yeah I thought we were going to add that frontend component to disable this call etc. for camp coordinators but since that work is scoped out, we can make this accessing for both admin and coordinators.

@TW1234567 could you update the ROLE comment and make this accessible by camp coordinators too?

Reply:
@TW1234567 Bump ^^

Reply:
Thanks for the reminder! Will get right to it.

Reply:
yeah let's remove the top level admin auth middleware since it's not necessarily universal anymore. Just give the camp coordinators to get all users. Ie: the GET endpoint for `localhost:5000/users/` only